test/integration/minikube_profile_test.go

```diff
@@ -0,0 +1,53 @@
+package integration
```

Comment:
This file is missing copyright boilerplate, you can copy it from another .go file and just change the date to 2022.